src/omnibar/omnibar.ts

```diff
@@ -554,6 +570,26 @@ function buildOmnibarUrl(): string {
   return omnibarUrl;
 }
 
+function updateTitle(): void {
+  let title = '';
```

Comment:
Since this is reassigned later, it doesn't need an initial value (`let title: string;`).